src/CitadelMinter.sol

```diff
@@ -172,6 +172,16 @@ contract CitadelMinter is
         gacPausable
         nonReentrant
     {
+        // Saves gas below if non-zero
+        uint256 cachedLockingBps = lockingBps;
+        uint256 cachedStakingBps = stakingBps;
+        uint256 cachedFundingBps = fundingBps;
+
+        require(
+            cachedLockingBps + cachedStakingBps + cachedFundingBps == MAX_BPS,
+            "CitadelMinter: Sum of propvalues must be 10000 bps"
+        );
```

Comment:
I don't think this require is necessary as the same check already happens within `setCitadelDistributionSplit` and this is the only way to alter these values.

Reply:
This is to ensure that `mintAndDistribute` is only called after `setCitadelDistributionSplit` has been called at least once